definitions/npm/prismic.io_v3.x.x/test_prismic.io_v3.x.x.js

```diff
@@ -0,0 +1,281 @@
+// @flow
```

Comment:
Can we [add a few `$ExpectError`](https://github.com/flowtype/flow-typed/blob/master/CONTRIBUTING.md#4-write-a-test-file-whose-name-starts-with-test_-in-the-package-version-directory) tests in here as well to sanity check that we get errors when we're supposed to?